src/CreateMikLabelModel/ML/DatasetHelper.cs

```diff
@@ -63,8 +63,13 @@ public async Task BreakIntoTrainValidateTestDatasetsAsync(string[] lines, string
             {
                 var totalCount = lines.Length;
 
+                const int MinIdealElementCount = 1_000;
                 // have at least 1000 elements
-                Debug.Assert(totalCount > 1000);
+                if (totalCount < MinIdealElementCount)
```

Comment:
Currently the MAUI repo has fewer than 1,000 issues labeled (it barely even has that many issues to begin with!), so I made this just a warning. (Otherwise Debug.Assert will crash the app.)